pandas/tests/io/parser/na_values.py

```diff
@@ -69,10 +69,7 @@ def test_non_string_na_values(self):
                 tm.assert_frame_equal(out, expected)
 
     def test_default_na_values(self):
-        _NA_VALUES = set(['-1.#IND', '1.#QNAN', '1.#IND', '-1.#QNAN',
-                          '#N/A', 'N/A', 'n/a', 'NA', '#NA', 'NULL', 'null',
-                          'NaN', 'nan', '-NaN', '-nan', '#N/A N/A', ''])
-        assert _NA_VALUES == parsers._NA_VALUES
+        _NA_VALUES = parsers._NA_VALUES
```

Comment:
I thought that assert is needed to make sure that the na_values constants are the same in both files, ' question: should the constants be repeated in na_values.py or reused from parsers(imported from common)
my understanding was that the real test is after these lines when dataframe is generated and tested that these values are replaced with NaN